dayPlanner.html

```diff
@@ -17,72 +17,73 @@
               </tr>
             </thead>
             <tbody>
-              <tr onclick="{clickableTR1()}">
+              <tr id="tr1" onclick="{funcGroup.func1()}">
                 <th>05:00 - 05:30</th>
-                <td></td> 
+                <td id="td1"></td> 

```

Comment:
- bütün todoların metinlerini `td` targetlerına atayabilmek için id tanımlıyoruz